src/python/knnPerfTest.py

```diff
@@ -33,60 +34,50 @@
 #
 # you may want to modify the following settings:
 
```

Comment:
Let's also update the doc string above with instructions on how to run this script with args?